fluent-plugin-webhdfs.gemspec

```diff
@@ -17,7 +17,8 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "rake"
   gem.add_development_dependency "test-unit"
-  gem.add_runtime_dependency "fluentd", '>= 0.10.53'
+  gem.add_development_dependency "appraisal"
+  gem.add_runtime_dependency "fluentd", '>= 0.10.59'
```

Comment:
Why this upgrade should be done? Are there any incompatibilities with 0.10.53 - 0.10.58?